admin-dev/themes/new-theme/js/app/pages/stock/components/header/stock-header.vue

```diff
@@ -60,6 +65,9 @@
       const toolbarButtons = getOldHeaderToolbarButtons();
       toolbarButtons.insertAfter($(this.$el).find('.title-row > .title'));
 
+      const notifications = getNotificationsElements();
+      notifications.insertAfter($(this.$el));
```

Comment:
If it's right and if this is possible, take care of the line 66, maybe it's not required too, I know it's not the PR scope, but if we can improve it... 

It's not the case, then just ignore my comment :)

Reply:
@NeOMakinG I'm affraid jQuery wrapper is required at line 66, since child elements are returned.
Alternative: declare a variable to optimize duplicated processing
```
      const $vueElement = $(this.$el);
      const toolbarButtons = getOldHeaderToolbarButtons();
      toolbarButtons.insertAfter($vueElement.find('.title-row > .title'));
      const notifications = getNotificationsElements();
      notifications.insertAfter($vueElement);
```

Reply:
I had a doubt too, this is why I said `If this is possible`

Your implementation could be the solution yeah